mne/viz/ica.py

```diff
@@ -400,7 +400,7 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, labels=None):
     exclude_labels = list()
     for ii in picks:
         if ii in exclude:
-            line_label = 'ICA %03d' % (ii + 1)
```

Comment:
:) no problem

On Tue, Sep 6, 2016 at 6:45 PM Andreas Højlund <notifications@github.com>
wrote:

> In mne/viz/ica.py
> <https://github.com/mne-tools/mne-python/pull/3565#discussion_r77672765>:
>
> > @@ -400,7 +400,7 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, labels=None):
> >      exclude_labels = list()
> >      for ii in picks:
> >          if ii in exclude:
> > -            line_label = 'ICA %03d' % (ii + 1)
>
> Sorry! Rookie mistake! Will upload in an hour or so.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3565/files/9e6fb2a09bb9abeed3a45c7c937bf70d9057b7d7#r77672765>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB0fim0D9VD3iCgtN0Do4m0-8e4q4vqgks5qnZiegaJpZM4J00Xp>
> .
>